src/applications/mhv-medications/api/rxApi.js

```diff
@@ -83,6 +84,38 @@ export const getPaginatedSortedList = (pageNumber = 1, sortEndpoint = '') => {
   );
 };
 
+export const getFilteredList = async filterOption => {
```

Comment:
do we need getPrescriptionSortedList below now?

Reply:
I don't think well need it once the filter feature goes to prod. since that list will take over. But atm, I think we have to keep it since its the only list that is getting live staging data, at least until the filter features development is done.

Reply:
I left comments to address the clean up after the filter feature is live. 👍